README.md

```diff
@@ -1,43 +1,36 @@
 # GOV.UK Display Screen
 
-Screen to display live data from GOV.UK. Includes number of people on GOV.UK,
-latest searches, trending and recent content. Not publicly accessible because
-there's sometimes personal data in the latest searches.
+Screen to display live data from GOV.UK. It displays the number of GOV.UK users in last 30 mins, live searches, 
+most popular pages and recent content. It is intended as an internal tool only because the live searches are unfiltered.
```

Comment:
There is some filtering for [email addresses, NI number](https://github.com/alphagov/govuk-display-screen/blob/main/public/javascripts/search.js#L23-L42). Maybe just have the first part of the sentence: 'It is intended as an internal tool only'.